README.md

```diff
@@ -1,28 +1,95 @@
+[![Build Status](https://github.com/boldlink/terraform-aws-dynamodb/actions/workflows/pre-commit.yml/badge.svg)](https://github.com/boldlink/terraform-aws-dynamodb/actions)
+
+[<img src="https://avatars.githubusercontent.com/u/25388280?s=200&v=4" width="96"/>](https://boldlink.io)
+
 # AWS DynamoDB Terraform module
 
 ## Description
 
-This terraform module creates a dynamodb table and items
+This terraform module creates a dynamodb table and item(s)
 
 Examples available [`here`](https://github.com/boldlink/terraform-aws-dynamodb/tree/main/examples)
 
+### **Important Points to Note**:
+- These examples use the latest version of this module
+- Global Secondary Index can be created or deleted for a previously created table. This can take more time if the table is populated.
+- Currently, there is no support for `ignore_changes` for both `write_capacity` and `read_capacity` in `global_secondary_index`.  
+An issue about this has been raised [here](https://github.com/hashicorp/terraform-provider-aws/issues/17096) hence it could be solved possibly when `global_secondary_index` will be a separate resource.
+- For Autoscaled indices (module `autoscaling_indexes`) it is important to have both `read_min_capacity` and `write_min_capacity` values match the values provided for `write_capacity` and `read_capacity` in the `global_secondary_index`. This will prevent terraform from trying to change the values when you do a `terraform plan/terraform apply`. See below;
+
+```hcl
+module "abc" {
+  ......(other configuration here)
+
+  autoscaling_indexes = {
+    IndexName = {
+      ......(other configuration here)
+
+      read_min_capacity  = 10
+      write_min_capacity = 10
+    }
+  }
+
+  global_secondary_index = [
+    {
+      ......(other configuration here)
+
+      write_capacity     = 10
+      read_capacity      = 10
+    }
+  ]
+}
+```
+
+## Usage
+
+```hcl
+module "dynamodb_table" {
+  source         = "../../"
+  name           = "minimum-example"
+  billing_mode   = "PROVISIONED"
```

Comment:
This example can be smaller, remember this is the absolute min settings you need to create a table

Reply:
@hugoalmeida264  In regards to the required values, I have further made it smaller